src/frontend/webpack.config.js

```diff
@@ -71,3 +72,28 @@ module.exports = {
     ],
   },
 };
+
+module.exports = (env, argv) => {
+  if (argv.mode === 'production') {
+    config.output = {
+      filename: 'index.js',
+      path: __dirname + '/../backend/marsha/static/js/build',
+      chunkFilename: '[name].[contenthash].js',
+      clean: true,
+    };
```

Comment:
As only `chunkFilename` property changes, we do not have to redefine the whole output object. From my point of view it improves readability and prevents mistake when output configuration will be updated.
```suggestion
    config.output.chunkFilename = '[name].[contenthash].js'
```